docs/actions/execute-javascript.md

```diff
@@ -0,0 +1,127 @@
+---
+id: execute-javascript-server-side
+title: Execute JavaScript (Server-side)
+sidebar_label: Execute JavaScript (Server)
+---
+
+> Audience: [`Citizen Developer`](/docs/audience#software-developers)<br/>
+> Skill Prerequisites: `JavaScript`
+
+Executes JavaScript code on the server side, optionally returning variables. This action is based on the [Microsoft/ClearScript](https://github.com/Microsoft/ClearScript) library that uses the [Google V8](https://v8.dev/) engine. It is a pure JavaScript framework, so there won't be access to the window or the document object. Other than that, it brings the full flexibility of JavaScript, plus some .net helpers to help write better code, faster.
+
+All tokens that exist in the Execution Context when this action runs will be available inside the script as local variables. Note that JavaScript is a case-sensitive language, so variables names will be in the exact case as the token name.
+
+## `Input Parameter Reference`
+
+| Parameter     | Description                           | Supports Tokens |
+|---------------|---------------------------------------|-----------------|
+| Script Name | A unique name that identifies what this script is about. It will be used for error and debug messages and logging and for as the filename for the `.js` file exported to GitHub| No |
```

Comment:
```suggestion
| Script Name | A unique name that identifies what this script is about. It will be used for error and debug messages and logging and as the filename for the `.js` file exported to GitHub| No |
```